index.js

```diff
@@ -12,4 +12,4 @@ require('./lib/locales');
 // locale data registered, and with English set as the default locale. Define
 // the `default` prop for use with other compiled ES6 Modules.
 exports = module.exports = IntlMessageFormat;
-Object.defineProperty(exports, 'default', {value: exports});
+exports['default'] = exports;
```

Comment:
I could conditionally use O.dP here if we prefer the `default` prop to be non-enum, etc. in ES5+ environments.